test/boulder-config.json

```diff
@@ -2,7 +2,8 @@
   "syslog": {
     "network": "",
     "server": "",
-    "stdoutlevel": 6
+    "stdoutlevel": 6,
+    "sysloglevel": 4
```

Comment:
Yeah, that was my bad - I agree with @ccppuu, we should leave this flag unspecified in the main boulder-config until it's in use in production.